inmem/auth_test.go

```diff
@@ -36,5 +37,6 @@ func initAuthorizationService(f platformtesting.AuthorizationFields, t *testing.
 }
 
 func TestAuthorizationService(t *testing.T) {
+	t.Skip("This service is not used, we use the kv inmem implementation")
```

Comment:
delete this file

Reply:
see https://github.com/influxdata/influxdb/pull/15784#discussion_r343671948